delivery/handlers/delivery_handlers.py

```diff
@@ -64,8 +64,11 @@ def post(self, staging_id):
                 self.reverse_url("delivery_status", delivery_id))
 
         self.set_status(ACCEPTED)
+
+        dds_version = dds_project.get_dds_version()
         self.write_json({'delivery_order_id': delivery_id,
-                         'delivery_order_link': status_end_point})
+                         'delivery_order_link': status_end_point,
+                         'dds_version':dds_version})
```

Comment:
```suggestion
                         'dds_version': dds_version})
```
Should be one space after `:` (see https://www.flake8rules.com/rules/E231.html)